index.html

```diff
@@ -54,6 +57,28 @@ <h1 class="popheading">Tonic</h1>
                       </div>
             </div>    
            </div>
+           <div class="containerdesktop">
+            <div class="imgdivdesktop">
+                <img class="tproject" class="projects" src="image1desktop.png" alt="Nature">
+            </div>
+            <div class="tonicdesktop">
+                    <h1 class="popheading popheaddesktop">Tonic</h1>
```

Comment:
- [ ] Kindly😃try to put here the correct margin between the sections to be like the Design and with that your portfolio will be nice for the visitors😊

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/215073596-e8ad6d1e-3966-4c9f-a0a0-8d5b903bcc1f.png) |    ![image](https://user-images.githubusercontent.com/83360636/215073642-835fd716-c295-4813-9f33-7180119b30aa.png)|

Reply:
suggested changes made and margin added. Screen shot attached for your kind reference below.
![image](https://user-images.githubusercontent.com/117091923/215099555-5d0135dd-a0f8-424c-975f-345f754cb16b.png)